chapter03/src/main/java/com/github/sebin/chapter03/Account.java

```diff
@@ -0,0 +1,87 @@
+package com.github.sebin.chapter03;
+
+public class Account {
+	private int accountID;
+	private User customer;
+	private String accountDescription;
+	private double accountBalance;
+	private double accountMaxBalance;
+	
+	
+	/**
+	 * @param accountID
+	 * @param customer
+	 * @param accountDescription
+	 * @param accountBalance
+	 * @param accountMaxBalance
+	 */
+	public Account(final int accountID,
+			final User customer,
+			final String accountDescription,
+			final double accountBalance,
+			final double accountMaxBalance) {
```

Comment:
final parameters? is this just to make sure these parameters will not be changed? but String and primitive types value won't be changed anyway, so I think maybe there are other usages here?